tensorflow/lite/micro/examples/mnist_lstm/train.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""
+LSTM model training for MNIST recognition
+
+Using python3:
+`python3 tensorflow/lite/micro/examples/mnist_lstm/train.py `
+
+Using bazel:
+`bazel build tensorflow/lite/micro/examples/mnist_lstm:train`
+`bazel-bin/tensorflow/lite/micro/examples/mnist_lstm/train`
+"""
+import argparse
```

Comment:
use absl instead of argparse for better compatibility.

https://github.com/tensorflow/tflite-micro/blob/659618e0a5221d9046e1bcb2f828b46568846499/tensorflow/lite/tools/strip_strings.py

Reply:
Done